data/categories.txt

```diff
@@ -1292,6 +1295,7 @@ fr:4Diététique
 hi:4स्वस्थ भोजन की दुकान|4हेल्थ फ़ूड स्टोर
 ru:Здоровая еда
 tr:Sağlıklı Gıda
+es:Alimentos saludables
```

Comment:
Let's sort this and other translations in the alphabet order if possible. Only en is always at the top.

Reply:
while adding words i saw that the whole file is out of order in several places, and it complicates the search for an specific language.
do you want to sort them all or just this added line?

IMHO a sort of the whole file is needed, but i prefer not to do it in this PR.

Reply:
Whole file can be sorted later, if no any other PRs are editing the same file.